release_notes/ocp-4-15-release-notes.adoc

```diff
@@ -1477,6 +1477,16 @@ In the following tables, features are marked with the following statuses:
 |====
 |Feature |4.13 |4.14 |4.15
 
+|Installing {product-title} on Oracle Cloud Infrastructure with VMs
+|N/A
+|Developer Preview
+|Technology Preview
+
+|Installing {product-title} on Oracle Cloud Infrastructure on baremetal
```

Comment:
```suggestion
|Installing {product-title} on {oci} on baremetal
```
Or if you'd prefer to have the Oracle name fully written out in this case for better clarity, you could probably use `{oci-first}` again.